README.md

```diff
@@ -33,7 +33,7 @@ let h_new = bcrypt::hash("password").unwrap();
 // Verify a password against an existing hash.
 let h = "$2y$05$bvIG6Nmid91Mu9RcmmWZfO\
          5HJIMCT8riNW0hEp8f6/FuA2/mHZFpe";
-assert_eq!(bcrypt::verify("password", h), true);
+assert!(bcrypt::verify("password", h));
```

Comment:
> Perhaps the example code with assert! would look nicer if "verify" was called "equals_password" or something like that

It certainly would, but I think that `equals_password` (or overriding `Eq`/`PartialEq`) isn't quite a correct way of describing that function: it doesn't test for literal, but for figurative equality, and in my experience it's better to be as literal as possible in your APIs.